code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -248,6 +250,19 @@
 		return FALSE
 	return TRUE
 
+/obj/item/robot_module/assistant
+	name = "Assistant"
+	basic_modules = list(
+		/obj/item/assembly/flash/cyborg,
+		/obj/item/reagent_containers/borghypo/epi,
+		/obj/item/weldingtool/largetank/cyborg,
+		/obj/item/crowbar/cyborg,
+		/obj/item/extinguisher)
```

Comment:
jesus christ, and I thought the PK borg's loadout was anemic

what the hell kind of a borg has only FIVE TOOLS (six if you count the speed boost), with FOUR of them being present in the engiborg module?

the speed boost is interesting, I guess, but it's really janky, especially with how dragging people while you have someone buckled to you as a borg works. tbh, it'd probably work better as just a passive, always-on speedboost.

Reply:
maybe work on a cargo borg, that sounds more like what you want here.

maybe also let borgs move crates that they're dragging to an empty tile that they click on like humans can? Not being able to do that is currently quite annoying, and probably is the main thing (other than spritework) stopping a cargo borg from being made.

Reply:
A basic cyborg whose main gimmick is both being a learner borg and moreseo focusing on direct human interaction.  Speedboost is a pretty strong ability, considering what you can do with it.  A borg of this vein is the only way I could see a "plain" borg being specialized, but I have better things to do.

Reply:
Also, the security borg also only had 5 tools by the end of its run.